packages/subproviders/test/integration/ledger_subprovider_test.ts

```diff
@@ -31,6 +42,10 @@ describe('LedgerSubprovider', () => {
             expect(accounts[0]).to.not.be.an('undefined');
             expect(accounts.length).to.be.equal(10);
         });
+        it('returns the expected account', async () => {
```

Comment:
`returns the expected first account from a ledger set up with the test mnemonic`

Reply:
If this is more descriptive, dev's who bump into it will understand the step them missed.